game/game.py

```diff
@@ -47,9 +47,14 @@
 class WumpusGame():
 
     def __init__(self) -> None:
+
+        # --------------------------------
```

Comment:
nit: this kind of comments aren't used in PRODUCTION environments (at Eventbrite)
```suggestion
```

Reply:
big mistake to leave these comments